ambry-messageformat/src/test/java/com/github/ambry/messageformat/MessageSievingInputStreamTest.java

```diff
@@ -581,6 +618,10 @@ private void testInValidDeletedAndExpiredBlobs(short blobVersion, BlobType blobT
         (int) messageFormatStream1.getSize() + (int) messageFormatStream2.getSize()
             + (int) messageFormatStream3.getSize() + (int) messageFormatStream4.getSize(),
         (int) messageFormatStream5.getSize());
+    corruptStream6.read(totalMessageStreamContent,
+        (int) messageFormatStream1.getSize() + (int) messageFormatStream2.getSize()
+            + (int) messageFormatStream3.getSize() + (int) messageFormatStream4.getSize()
+            + (int) messageFormatStream5.getSize(), (int) messageFormatStream6.getSize());
```

Comment:
minor: we have lots of redundant calculation of the offset in `read` method, suggest simplifying them.